docs/buildah-bud.md

```diff
@@ -497,48 +501,6 @@ the user namespace in which `buildah` itself is being run should be reused, or
 it can be the path to an user namespace which is already in use by another
 process.
 
-**--userns-uid-map** *mapping*
```

Comment:
This is making me nervous?
Something is going wrong here.
```
$ podman build --help | grep userns
      --userns path                                'container', path of user namespace to join, or 'host'
      --userns-gid-map containerID:hostID:length   containerID:hostID:length GID mapping to use in user namespace
      --userns-gid-map-group name                  name of entries from /etc/subgid to use to set user namespace GID mapping
      --userns-uid-map containerID:hostID:length   containerID:hostID:length UID mapping to use in user namespace
      --userns-uid-map-user name                   name of entries from /etc/subuid to use to set user namespace UID mapping
```
These options are showing up in podman build but not buildah bud.

Reply:
@nalind @TomSweeneyRedHat @baude Any ideas?

Reply:
OK I think I have it, the issue is that these flags are global flags not just for podman.

```
./bin/buildah --help | grep userns
      --userns-gid-map ctrID:hostID:length   default ctrID:hostID:length GID mapping to use
      --userns-uid-map ctrID:hostID:length   default ctrID:hostID:length UID mapping to use
```

Reply:
So we could move this content to the buildah man page.  But I think we should still link it from here.

Reply:
It already is in the buildah man page: https://github.com/containers/buildah/blame/master/docs/buildah.md#L70-L88

Reply:
I can, if so desired, add back this section and add a special case for the xref script to ignore it. That was my original question on my initial commit; since I never heard from anyone, I just made an executive decision. If I made the wrong one, I'm happy to revert it.

Reply:
I'll dive more tomorrow, too much eyestrain at the moment, but this removal doesn't feel right.

Reply:
I think we should take this section and copy it into the buildah man page.  There is a lot more useful information here on configuring user namespace then is currently in the buildah man page.  I think we should also update the two userns-uid-map-user sections to refernence back to this in the main man page.  Sort of a see also --userns-uid-map in buildah(1) man page.

Reply:
I opened https://github.com/containers/buildah/pull/2661 to move the man page description around.